src/ui/linux/TogglDesktop/main.cpp

```diff
@@ -10,6 +10,12 @@
 
 #include <stdint.h>
 #include <stdbool.h>
+#include <stdio.h>
+#include <iostream>
+#include <unistd.h>
+
+#include "Poco/ErrorHandler.h"
```

Comment:
This gives me error: 
```
fatal error: Poco/ErrorHandler.h: no such file or directory
```

Nowhere in linux project has poco been referenced like this. We've only referenced Poco in library. I think it's not possible to reference Poco this way. Maybe someway through library.

Reply:
hmh, for some reason it does compile for me, i just forgot to remove this from when i tested stuff around poco errors - i'll remove it right away